oci/spec_opts.go

```diff
@@ -607,7 +607,7 @@ func WithUser(userstr string) SpecOpts {
 		// The `Username` field on the runtime spec is marked by Platform as only for Windows, and in this case it
 		// *is* being set on a Windows host at least, but will be used as a temporary holding spot until the guest
 		// can use the string to perform these same operations to grab the uid:gid inside.
```

Comment:
Can you add a blurb on this comment stating basically what's in the commit description, e.g. "NOTE: Mounts aren't supported on Darwin, so lets do the same" or something similar.